config.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+/* eslint no-process-env: 0 */
+module.exports = {
+  sql: {
```

Comment:
As you only currently support mysql this may as well be `mysql` rather than `sql`.

Reply:
fair